### DISCUSSION TITLE
Is the authentification of the workers protected without TLS

### DISCUSSION THREAD

**Discussion by Try2Code** [buildbot/buildbot#6823] [General]
hi folks!
Some admins have security concerns regarding use the workers without tls switched on. Unfortunately I cannot make that work at the moment, but since the payload (change sets, status reports,logs) are harmless, the only thing left is the authentication of the workers itself.

Hence the questions: Is the authentication protected even without TLS?

thx in advance

**Comment by vibbo** [comment]
Hi,

AFAIK none of the communication is protected when TLS is not used.

I am using TLS, but I have to agree, that it is not very well documented. Especially, when in the end, it is not that complicated to setup.

I am planning to submit some updates to docs and code to make it easier.

Cheers

**Reply by Try2Code** [reply]
thx, that would be great!

**Comment by p12tic** [comment]
The authentication data itself is protected as plain credentials are not sent over the wire.

**Reply by Try2Code** [reply]
thx  a lot @p12tic
